data/sql/updates/pending_db_world/Fix_quest_11879.sql

```diff
@@ -0,0 +1,51 @@
+UPDATE `creature_template` SET `unit_flags` = 770 WHERE (`entry` = 25802);
+-- Kaw speach on event start
+DELETE FROM `creature_text` WHERE `CreatureID`= 25802;
+INSERT INTO `creature_text` (`CreatureID`, `GroupID`, `ID`, `Text`, `Type`, `Language`, `Probability`, `Emote`, `Duration`, `Sound`, `BroadcastTextId`, `TextRange`, `comment`) VALUES 
+(25802, 0, 0, 'You challenge Kaw, destroyer of mammoths? Then face me and feel my thunder!', 14, 0, 100, 0, 0, 0, 25071, 0, 'Kaw the Mammoth Destroyer'),
+(25802, 1, 0, 'Kaw\'s halberd breaks in two and falls to the ground!', 41, 0, 100, 0, 0, 930, 25087, 0, 'Kaw the Mammoth Destroyer Death');
+-- Kaw SAI
+DELETE FROM `smart_scripts` WHERE (`source_type` = 0 AND `entryorguid` = 25802);
+INSERT INTO `smart_scripts` (`entryorguid`, `source_type`, `id`, `link`, `event_type`, `event_phase_mask`, `event_chance`, `event_flags`, `event_param1`, `event_param2`, `event_param3`, `event_param4`, `event_param5`, `event_param6`, `action_type`, `action_param1`, `action_param2`, `action_param3`, `action_param4`, `action_param5`, `action_param6`, `target_type`, `target_param1`, `target_param2`, `target_param3`, `target_param4`, `target_x`, `target_y`, `target_z`, `target_o`, `comment`) VALUES
+(25802, 0, 0, 0, 101, 0, 100, 1, 1, 25, 0, 0, 0, 0, 80, 2580200, 2, 0, 0, 0, 0, 1, 0, 0, 0, 0, 0, 0, 0, 0, 'Kaw the Mammoth Destroyer - On Min Players in Range - Run Script (No Repeat)'),
+(25802, 0, 1, 0, 6, 0, 100, 0, 0, 0, 0, 0, 0, 0, 11, 46310, 2, 0, 0, 0, 0, 1, 0, 0, 0, 0, 0, 0, 0, 0, 'Kaw the Mammoth Destroyer - On Just Died - Cast \'Drop War Halberd\''),
+(25802, 0, 2, 0, 6, 0, 100, 1, 0, 0, 0, 0, 0, 0, 1, 1, 1000, 0, 0, 0, 0, 1, 0, 0, 0, 0, 0, 0, 0, 0, 'Kaw the Mammoth Destroyer - On just died - Alert Message');
+-- Kaw SAI Timed Actionlists
+DELETE FROM `smart_scripts` WHERE (`entryorguid` = 2580200) AND (`source_type` = 9);
+INSERT INTO `smart_scripts` (`entryorguid`, `source_type`, `id`, `link`, `event_type`, `event_phase_mask`, `event_chance`, `event_flags`, `event_param1`, `event_param2`, `event_param3`, `event_param4`, `event_param5`, `event_param6`, `action_type`, `action_param1`, `action_param2`, `action_param3`, `action_param4`, `action_param5`, `action_param6`, `target_type`, `target_param1`, `target_param2`, `target_param3`, `target_param4`, `target_x`, `target_y`, `target_z`, `target_o`, `comment`) VALUES 
+(2580200, 9, 1, 0, 0, 0, 100, 1, 0, 0, 0, 0, 0, 0, 59, 1, 0, 0, 0, 0, 0, 1, 0, 0, 0, 0, 0, 0, 0, 0, 'Kaw - Timed - Enable Running'),
+(2580200, 9, 3, 0, 0, 0, 100, 1, 2500, 2500, 0, 0, 0, 0, 11, 46260, 2, 0, 0, 0, 0, 19, 25881, 100, 0, 0, 0, 0, 0, 0, 'Kaw - Timed - Mount to Moria'),
+(2580200, 9, 2, 0, 0, 0, 100, 1, 3000, 3000, 0, 0, 0, 0, 69, 0, 0, 0, 0, 0, 0, 8, 0, 0, 0, 0, 3974.17, 5476.31, 35.602, 5.564, 'Kaw - Timed - Move to Moria'),
+(2580200, 9, 0, 0, 0, 0, 100, 1, 0, 0, 0, 0, 0, 0, 1, 0, 2000, 0, 0, 0, 0, 1, 0, 0, 0, 0, 0, 0, 0, 0, 'Kaw - Timed - Yell');
```

Comment:
Entry: 25802 spawns with flags 33024 (Weird that he doesn't have 512, but it seems blizz)
Entry: 25881 spawns with flags 33024 as well

Note that when you have to change flags in the script. It also needs to be set on spawn, otherwise it can have some issues with resets. So their templates should have 32768 and 256 is added On Respawn

Hit AreaTrigger ->
 - Yell Line
 - Wait 2400, Emote 53
 - Wait 3000 Run to X: 3973.9749 Y: 5477.6187 Z: 35.615986

On Reached ->
 - Cast 43671 on vehicle 25881
 - Remove Immune Flags from both

Reply:
> Entry: 25802 spawns with flags 33024 (Weird that he doesn't have 512, but it seems blizz) Entry: 25881 spawns with flags 33024 as well
> 
> Note that when you have to change flags in the script. It also needs to be set on spawn, otherwise it can have some issues with resets. So their templates should have 32768 and 256 is added On Respawn
> 
> Hit AreaTrigger ->
> 
> * Yell Line
> * Wait 2400, Emote 53
> * Wait 3000 Run to X: 3973.9749 Y: 5477.6187 Z: 35.615986
> 
> On Reached ->
> 
> * Cast 43671 on vehicle 25881
> * Remove Immune Flags from both

Thanks for your help Gul, I'll solve this (or at least try) haha 😄

Reply:
@Gultask it wouldn't be flag 33536?

Reply:
> Entry: 25802 spawns with flags 33024 (Weird that he doesn't have 512, but it seems blizz) Entry: 25881 spawns with flags 33024 as well
> 
> Note that when you have to change flags in the script. It also needs to be set on spawn, otherwise it can have some issues with resets. So their templates should have 32768 and 256 is added On Respawn
> 
> Hit AreaTrigger ->
> 
> * Yell Line
> * Wait 2400, Emote 53
> * Wait 3000 Run to X: 3973.9749 Y: 5477.6187 Z: 35.615986
> 
> On Reached ->
> 
> * Cast 43671 on vehicle 25881
> * Remove Immune Flags from both

@Gultask  it wouldn't be flag 33536?

Reply:
33536 would be better imo, but add in the comment like "Add Immunity Flags, Sniffed: 33024"
but you can keep 2 out, we never actually use it